azure-pipelines.yml

```diff
@@ -196,7 +196,7 @@ stages:
             versionSpec: $(NodeJSVersion)
 
         - task: UseDotNet@2
-          displayName: Add dotnet
+          displayName: Use .NET SDK
```

Comment:
Some say `Use .NET SDK` and others `Use .NET SDK 5.0`.  I'd prefer simply `Use .NET SDK` since the version will eventually change again, but ultimately I don't care too much.